lua/entities/gmod_wire_gpu/cl_init.lua

```diff
@@ -326,6 +326,9 @@ function ENT:Draw()
 
   -- Draw GPU itself
   self:DrawModel()
+  
+	local tone = render.GetToneMappingScaleLinear()
+	render.SetToneMappingScaleLinear(VECTOR_1_1_1)
```

Comment:
There's a 'return' in this function that will cause the toneMapping to get stuck as 1,1,1. Need to find a way to remove the return.

Reply:
Can easily make the rest of the function as part of an else statement.